utils_cv/tracking/model.py

```diff
@@ -104,22 +206,18 @@ def savetxt_results(results: Dict[int, List[TrackingBbox]],
     Returns:
         result_path: saved prediction results txt file path
     """
-    if not root_path:
-        with tempfile.TemporaryDirectory() as tmpdir1:
-            os.makedirs(osp.join(tmpdir1, exp_name))
-            result_path = osp.join(tmpdir1, exp_name, result_filename)
-    else:
-        result_path = osp.join(root_path, exp_name, result_filename)
 
-    # Save results in MOT format for evaluation            
+    # Convert prediction results to mot format
     bboxes_mot = boxes_to_mot(results)
+
+    # Save results
+    result_path = osp.join(root_path, exp_name, result_filename)
```

Comment:
I believe this assume "root_path" is initialized, however root_path if not specified is None. That said, maybe it still works also if root_path is None since the osp.join() just ignores it? If so that'd be fine then.